.github/workflows/rundata_check.yml

```diff
@@ -0,0 +1,12 @@
+name: Rundata Check
```

Comment:
While it's true that all the checks could be merged under pull_request.yml, I find it hard to edit because the file is getting long, and YAML does not navigate easily in a text editor, without using a specialized YAML editor.

The disadvantage is that I need to add the new rundata_check.yml to rebase_check.yml, and I forgot to do that.